website/docs/r/network_direct.html.markdown

```diff
@@ -0,0 +1,33 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_network_direct"
+sidebar_current: "docs-vcd-resource-network-direct"
+description: |-
+  Provides a vCloud Director VDC Network attached to an external one. This can be used to create, modify, and delete internal networks for vApps to connect.
+---
+
+# vcd\_network\_direct (*v2.0+*)
+
+Provides a vCloud Director VDC Network directly connected to an external network. This can be used to create,
+modify, and delete internal networks for vApps to connect.
+
+## Example Usage
+
+```hcl
+resource "vcd_network_direct" "net" {
+  org              = "my-org"
+  vdc              = "my-vdc"
```

Comment:
More people will use the adapter with org admin, as opposed to sys admin. So, let's mark `org` and `vdc` as optional and separate them by a new line (in all three examples). For example like this:

```
resource "vcd_network_direct" "net" {
  org              = "my-org" # Optional
  vdc              = "my-vdc" # Optional

  name             = "my-net"
  external_network = "my-ext-net"
}
```

Reply:
done